Dockerfile.deploy

```diff
@@ -55,7 +64,13 @@ RUN pip install --no-cache-dir --exists-action=w --no-deps -r requirements/syste
     && pip install --no-cache-dir --exists-action=w --no-deps -r requirements/prod_without_hash.txt\
     && pip install --no-cache-dir --exists-action=w --no-deps -e .
 
-RUN echo -e "from olympia.lib.settings_base import *\n\
+# Link /usr/sbin/uwsgi and /usr/bin/uwsgi to deal with migration from Centos -> Debian
+RUN ln -s /usr/bin/uwsgi /usr/sbin/uwsgi
+
+# Fix for https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=783738
+RUN ln -s /usr/lib/python2.7/plat-*/_sysconfigdata_nd.py /usr/lib/python2.7/
```

Comment:
Instead of the wildcard *, is it possible to be specific of the directory name, or is it because there are multiple directories involved?

Reply:
There are multiple files involved.

Reply:
There are multiple files being linked.